docs/api-reference/supervisor-api.md

```diff
@@ -23,89 +23,3075 @@ sidebar_label: Supervisors
+
+#### Responses
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--200 SUCCESS-->
+<br/>
+*Successfully retrieved array of active Supervisor IDs* 
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+---
+
+#### Sample request
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cURL-->
+```shell
+curl "http://<ROUTER_IP>:<ROUTER_PORT>/druid/indexer/v1/supervisor"
+```
+<!--HTTP-->
+```HTTP
+GET /druid/indexer/v1/supervisor HTTP/1.1
+Host: http://<ROUTER_IP>:<ROUTER_PORT>
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+#### Sample response
+
+<details>
+  <summary>Click to show sample response</summary>
+  
+  ```json
+  [
+    "wikipedia_stream",
+    "social_media"
+  ]
+  ```
+</details>
+
+### Get an array of active Supervisor objects
+
+#### URL
+
+<code className="getAPI">GET</code> `/druid/indexer/v1/supervisor?full`
+
+Retrieves an array of active Supervisor objects. Each object has properties relevant to the Supervisor: `id`, `state`, `detailedState`, `healthy`, and `spec`. 
+
+See the following table for details on properties within the response object:
 
 |Field|Type|Description|
 |---|---|---|
-|`id`|String|supervisor unique identifier|
-|`state`|String|basic state of the supervisor. Available states:`UNHEALTHY_SUPERVISOR`, `UNHEALTHY_TASKS`, `PENDING`, `RUNNING`, `SUSPENDED`, `STOPPING`. Check [Kafka Docs](../development/extensions-core/kafka-supervisor-operations.md) for details.|
-|`detailedState`|String|supervisor specific state. See documentation of specific supervisor for details: [Kafka](../development/extensions-core/kafka-ingestion.md) or [Kinesis](../development/extensions-core/kinesis-ingestion.md)|
-|`healthy`|Boolean|true or false indicator of overall supervisor health|
-|`spec`|SupervisorSpec|JSON specification of supervisor|
+|`id`|String|Supervisor unique identifier|
+|`state`|String|The basic state of the supervisor. Available states:`UNHEALTHY_SUPERVISOR`, `UNHEALTHY_TASKS`, `PENDING`, `RUNNING`, `SUSPENDED`, `STOPPING`. Check [Kafka Docs](../development/extensions-core/kafka-supervisor-operations.md) for details.|
+|`detailedState`|String|The Supervisor specific state. See documentation of specific supervisor for details: [Kafka](../development/extensions-core/kafka-ingestion.md) or [Kinesis](../development/extensions-core/kinesis-ingestion.md)|
+|`healthy`|Boolean|True or false indicator of overall Supervisor health|
+|`spec`|SupervisorSpec|JSON specification of the Supervisor|
+
+#### Responses
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--200 SUCCESS-->
+<br/>
+*Successfully retrieved active Supervisor objects* 
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+---
+
+#### Sample request
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cURL-->
+```shell
+curl "http://<ROUTER_IP>:<ROUTER_PORT>/druid/indexer/v1/supervisor?full=null"
+```
+<!--HTTP-->
+```HTTP
+GET /druid/indexer/v1/supervisor?full=null HTTP/1.1
+Host: http://<ROUTER_IP>:<ROUTER_PORT>
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+#### Sample response
```

Comment:
The supervisor objects in the response body contain the `suspended` property, which is not listed in the table. Instead of having two separate tables documenting the same fields, consider adding a paragraph listing out supervisor object properties following the Supervisor information heading. For example:

```
## Supervisor information
The following table lists the properties of the supervisor object:
(list all of the properties here)
```